presto-testing/src/main/java/io/prestosql/testing/AbstractTestQueryFramework.java

```diff
@@ -325,7 +325,12 @@ protected void assertTableColumnNames(String tableName, String... columnNames)
 
     protected void assertExplainAnalyze(@Language("SQL") String query)
     {
-        String value = (String) computeActual(query).getOnlyValue();
+        assertExplainAnalyze(getSession(), query);
+    }
+
+    protected void assertExplainAnalyze(Session session, @Language("SQL") String query)
```

Comment:
as `assertExplainAnalyze(String)` brings very little benefit, let's not add an overload to it.

instead, in the actual test, just call `computeActual` and run `assertThat` on the returned `String`

(and computeActual already accepts a session)